### ISSUE TITLE
Dropdown not working since .text was deprecated 

### ISSUE THREAD

**Issue by parkerdonat** [hackiftekhar/IQDropDownTextField#36]
I changed my .text to .selectedItem and now the dropdown menu doesn't activate when tapped anymore. I checked this on my device and on simulator. Here is my sample code to initiate the dropdown: 

```
-(void)textFieldDidBeginEditing:(UITextField *)textField {
    [self.messageField becomeFirstResponder];
    self.messageField.tintColor = [UIColor redColor];

     if ([textField.placeholder isEqualToString:@"Your message"]) {
      self.messageField.placeholder = @"";
      self.previewLabel.text = self.messageField.text;
    } else if (textField == self.messageField){
         [self.previewLabel setFont:[UIFont fontWithName:self.fontTextField.selectedItem size:[self.sizeTextField.selectedItem floatValue]]];
        self.previewLabel.text = self.messageField.text;
    }
}
```

Any ideas on how to fix this? Any help would be greatly appreciated. Thanks!

**Comment by myusuf08** [comment]
how to fix it?

**Comment by hackiftekhar** [comment]
How you are setting your textField 'itemList'? What was your 'isOptional' parameter? Could you please share a demo?

**Comment by parkerdonat** [comment]
Sure. So I set these in the viewDidLoad. Here is an example of how I set them. 

self.fontTextField.isOptionalDropDown = NO;
    self.fontTextField.delegate = self;
    [self.fontTextField setItemList:[UIFont arrayOfFonts]];

**Comment by hackiftekhar** [comment]
Don't know why but it should work. Could you please share a demo so that I can tKe a look at weekend.

**Comment by hackiftekhar** [comment]
Could you please provide a demo?

**Comment by hackiftekhar** [comment]
I believe this is fixed with #50, could you please recheck?
